pom.xml

```diff
@@ -163,7 +163,6 @@
 
         <osgi-resource-locator.version>1.0.3</osgi-resource-locator.version>
         <snakeyaml.version>2.2</snakeyaml.version>
-        <jakarta.servlet.jsp.jstl.version>3.0.1</jakarta.servlet.jsp.jstl.version>
```

Comment:
You also need to remove/replace this here: https://github.com/payara/Payara/blob/5f3c6cbf93e1fc490b6de6c9519b287748a3f755/appserver/pom.xml#L279